celesta-sql/src/test/java/ru/curs/celesta/score/ParserTest.java

```diff
@@ -308,17 +306,20 @@ public void test4() throws Exception {
     ForeignKey fk = t.getForeignKeys().iterator().next();
     assertEquals("fk_x_rolempyees_xroles", fk.getConstraintName());
 
+    assertTrue(g.getLength() > 10000);
+    assertTrue(g.getChecksum() != 0);
 
     //TODO: non-stable values when using Git because of line ending conversion
     //hash sum calculation should be rewritten to be line-ending independent
+    //recalculation of assert expected values is needed
     if (";".equals(File.pathSeparator)){
       //Windows!
-      assertEquals(20767, g.getLength());
-      assertEquals(0x1754E6E7, g.getChecksum());
+//      assertEquals(20767, g.getLength());
```

Comment:
why?

Reply:
The outcome of this test depends on local Git configuration. The file can be checked out both with Linux and Windows style EOLs, and sometimes the test fails.

Reply:
So...let's remove it!

Reply:
@inponomarev So shall we remove the out-commented lines?

Reply:
do as you wish, I don't think this matters anything